.editorconfig

```diff
@@ -0,0 +1,32 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+end_of_line = lf
```

Comment:
🤔 As a separate PR, it might be good to configure git to not convert `\n` to `\n\r`on windows via a `.gitattributes` file.

I have noticed that this can help Windows and Mac users develop more efficiently together, and avoid mysterious errors when ultimately running the code in a linux container. @ermish, since you use Windows, what do you think?

Reply:
I like the idea, but I'm not sure this is incompatible with Windows....I opened the branch on my windows machine and I'm getting a million errors running `yarn lint`. Do you get these as well? Haven't tried on my macbook yet

![image](https://user-images.githubusercontent.com/4411499/213684497-f728186d-f8ab-4250-bc96-de094dc27a18.png)

Reply:
hmmm...I'm going to test your idea for a .gitattributes file and see if this solves it

Reply:
ok, I think the gitattributes won't auto fix it, but it will fix when new files are downloaded/cloned. 
I found a vscode option to use `/n` by default on windows, so we should be good! ship it 💯

Reply:
we should add the `.gitattributes` in another PR like you recommended for future contributors 😸